### ISSUE TITLE
Google meet not working

### ISSUE THREAD

**Issue by areenraj** [getferdi/ferdi#1250]
Hey, so I was using trying to use google meet for my classes and everytime I try to connect to a meeting, the camera starts but a blank screen appears. 
![Screenshot from 2021-03-19 12-40-16](https://user-images.githubusercontent.com/80944803/111743875-549b0800-88b0-11eb-90eb-8af1b7b36e86.png)
Is there any fix to this?

**Comment by Andre-Oliveira** [comment]
I have the same issue, by opening the Developer tools i can see a few messages saying `Uncaught (in promise) Error: DisconnectedError, EndCause = 16` and `m=_b,_tp:151 Uncaught (in promise) _.ng {message: "Signal initially disposed"`

**Comment by vraravam** [comment]
With the latest version of Ferdi and the latest recipes, this should no longer be the case.

Can you please try to upgrade to the nightlies available [here](https://github.com/getferdi/nightlies/releases) AND then upgrading the recipes by following the steps outlined [here](https://github.com/getferdi/ferdi#troubleshooting-recipes-self-help)?

If you are on macos, I have shared some newer builds of the latest source code [here](https://drive.google.com/drive/folders/1mJmKbhpVHdb066IHlrdYiD8TG9wlmuBv)

**Comment by Cydonia7** [comment]
Hello, I'm experiencing the same issue on Ferdi 5.6.1-nightly.7 with upgraded recipe.

**Comment by vraravam** [comment]
I have this working (not sure which version it was fixed as of) - but, I'm on the latest `5.6.1-beta.1` and its working for me on macos big sur. Closing since there have not been any new updates to the question asked.

**Comment by areenraj** [comment]
@vraravam @Andre-Oliveira @Cydonia7 
I finally got it to work in the Ferdi 5.6.3 nightly build. 
I upgraded Ferdi and replaced the recipes folder but still nothing happened. I was getting the error that meet does not work in my browser. 
Then I clicked on 'Edit Service' and found the user agent stream. Googled what my user agent was and found out it to be - "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.82 Safari/537.36". I entered it into the field and reloaded the service and BOOM it works. 
Its kind of dumb luck because I really don't know what I am working with here but I hope this helps all of you. 
![Screenshot from 2021-09-19 12-13-29](https://user-images.githubusercontent.com/80944803/133918241-98ffed62-4655-4058-b349-1b219cfcc119.png)
P.S. -The reason I didn't bother to come back to this issue because I stopped using Ferdi (due to the inability to access Meet). But its nice to see that a lot of problems (like Drive not working) have been fixed.
